src/doc/rustc/src/platform-support.md

```diff
@@ -416,25 +416,7 @@ target | std | host | notes
 [`xtensa-esp32s2-espidf`](platform-support/esp-idf.md) | ✓ |  | Xtensa ESP32-S2
 [`xtensa-esp32s2-none-elf`](platform-support/xtensa.md) | * |  | Xtensa ESP32-S2
 [`xtensa-esp32s3-espidf`](platform-support/esp-idf.md) | ✓ |  | Xtensa ESP32-S3
-<<<<<<< HEAD
```

Comment:
What happened here? Did upstream remove all these?

Reply:
they moved them to other places

Reply:
Whoa, cool